src/components/table.tsx

```diff
@@ -2,7 +2,8 @@ import { VNode, render } from "preact";
 import { csvFormatValue } from "d3-dsv";
 import Icon from "./icon";
 
-export type TableCell = string | number | VNode;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Per prior point, if we can just remove this rule override in the ESLint config, we wouldn't need to disable the rule.

Reply:
gone: 5fe1f51aa670f485f41791c65368114dad6da057